settings.xml

```diff
@@ -0,0 +1,13 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                      http://maven.apache.org/xsd/settings-1.0.0.xsd">
+
+    <servers>
+        <server>
+            <id>eblocker-top</id>
+            <username>NoUserName</username>
+            <password>&#103;hp_Cei0Aao5u77MAOxJu7Hlwc1piSERTk3rv8Pt</password>
```

Comment:
The token must not be inserted directly here. It must first be encoded with e.g. "docker run jcansdale/gpr encode TOKEN". If this is not done, the token in github will be deleted immediately.